src/components/Alteration/Articles/ResolutionDates.vue

```diff
@@ -86,8 +86,8 @@
           title="Resolution or Court Order Date"
           nudge-right="80"
           nudge-top="15"
-          :minDate="getBusinessFoundingDate"
```

Comment:
This was in API format, not yyyy-mm-dd as expected.

Now using getters to convert this correctly.